python/dask_cudf/dask_cudf/tests/test_applymap.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+import pytest
+from pandas import NA
+
+from dask import dataframe as dd
+
+from .utils import _make_random_frame
```

Comment:
Can we do an absolute import here instead of a relative import so that it is consistent with other imports here and elsewhere in the code-base?